pinot-controller/src/main/java/org/apache/pinot/controller/api/access/BasicAuthAccessControlFactory.java

```diff
@@ -50,8 +53,9 @@
 
```

Comment:
Similar to Broker auth, this should be in its own class for backwards-compatibility.